dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/ZenodoHelperTest.java

```diff
@@ -44,4 +47,58 @@ public void extractDoiFromBadDoiUrl() {
         Assert.assertNotEquals("10.5072/zenodo.372767", doi);
     }
 
+    @Test
+    public void checkAliasCreationFromDoiWithInvalidPrefix() {
+        User user = new User();
+        user.setIsAdmin(false);
+        user.setCurator(false);
+        String doi = "drs:10.5072/zenodo.372767";
+        boolean throwsError = false;
+        try {
+            String formattedDoi = ZenodoHelper.createAliasUsingDoi(doi, user);
+        } catch (CustomWebApplicationException ex) {
+            throwsError = true;
+        }
+
+        if (!throwsError) {
+            fail("Non admin or curator was able to create an alias with an invalid prefix.");
+        }
+    }
+
+    @Test
+    public void checkAdminAliasCreationFromDoiWithInvalidPrefix() {
+        User user = new User();
+        user.setIsAdmin(true);
+        user.setCurator(false);
+        String doi = "drs:10.5072/zenodo.372767";
+        boolean throwsError = false;
+        try {
+            String formattedDoi = ZenodoHelper.createAliasUsingDoi(doi, user);
+        } catch (CustomWebApplicationException ex) {
+            throwsError = true;
+        }
+
+        if (throwsError) {
+            fail("Admin was not able to create an alias with an invalid prefix.");
+        }
+    }
+
+    @Test
+    public void checkNonAdminAliasCreationFromValidDoi() {
+        User user = new User();
+        user.setIsAdmin(false);
+        user.setCurator(false);
+        String doi = "10.5072/zenodo.372767";
+        boolean throwsError = false;
+        try {
+            String formattedDoi = ZenodoHelper.createAliasUsingDoi(doi, user);
+        } catch (CustomWebApplicationException ex) {
+            throwsError = true;
+        }
+
+        if (throwsError) {
```

Comment:
Shouldn't this be `!throwsError`?

Reply:
No because the input is a valid DOI to use to create an alias